api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestSingleShot.java

```diff
@@ -765,7 +764,7 @@ public void testNNFWTFLite() {
 
                 /* check output */
                 float expected = i + 3.5f;
-                assertTrue(expected == output.getTensorData(0).getFloat(0));
+                assertEquals(expected, output.getTensorData(0).getFloat(0), 0.0);
```

Comment:
For float, wouldn't be be appropriate to add non-zero ranges? (e.g., ```s/0.0/0.0001```?)

Reply:
I updated this delta (0.0 -> 0.0f)